f1tenth_racecar/launch/known_map_localization.launch

```diff
@@ -1,7 +1,7 @@
+<?xml version="1.0"?>
```

Comment:
This file is also deprecated since we're no longer using acml.
Could probably get rid of more dependencies.

Reply:
Removed.